frb_dart/lib/src/basic.dart

```diff
@@ -64,29 +65,36 @@ abstract class FlutterRustBridgeBase<T extends FlutterRustBridgeWireBase> {
 
   /// Similar to [executeNormal], except that this will return a [Stream] instead of a [Future].
   @protected
-  Stream<S> executeStream<S>(FlutterRustBridgeTask<S> task) async* {
+  Stream<S> executeStream<S, E extends Object>(
+      FlutterRustBridgeTask<S, E> task) async* {
     final receivePort = ReceivePort();
     task.callFfi(receivePort.sendPort.nativePort);
 
     await for (final raw in receivePort) {
       try {
-        yield _transformRust2DartMessage(raw, task.parseSuccessData);
+        yield _transformRust2DartMessage(
+            raw, task.parseSuccessData, task.parseErrorData);
       } on _CloseStreamException {
         receivePort.close();
       }
     }
   }
 
-  S _transformRust2DartMessage<S>(
-      dynamic raw, S Function(dynamic) parseSuccessData) {
+  S _transformRust2DartMessage<S, E extends Object>(
+      dynamic raw,
+      S Function(dynamic) parseSuccessData,
+      E Function(dynamic)? parseErrorData) {
     final action = raw[0];
     switch (action) {
       case _RUST2DART_ACTION_SUCCESS:
         assert(raw.length == 2);
         return parseSuccessData(raw[1]);
       case _RUST2DART_ACTION_ERROR:
-        assert(raw.length == 4);
-        throw FfiException(raw[1], raw[2], raw[3]);
+        assert(raw.length == 5);
```

Comment:
Why length 5 - I guess maybe only 2 entries?

Reply:
👀 It was resolved without comments or code changes

Reply:
I was going to send the commit and forgot